patches/@types+lodash+4.14.194.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/node_modules/@types/lodash/index.d.ts b/node_modules/@types/lodash/index.d.ts
+index 8d1ec9f..0d4b05f 100755
+--- a/node_modules/@types/lodash/index.d.ts
++++ b/node_modules/@types/lodash/index.d.ts
+@@ -31,15 +31,3 @@ declare namespace _ {
+     // tslint:disable-next-line no-empty-interface (This will be augmented)
+     interface LoDashStatic {}
+ }
+-
+-// Backward compatibility with --target es5
+-declare global {
```

Comment:
not sure why lodash has this but I don't think it's needed and it was causing our build to fail because we don't skip lib check